### DISCUSSION TITLE
Aggregates

### DISCUSSION THREAD

**Discussion by pan-zhuo** [crystalyragui/MARC2RDA#354] [General]
What are the 'default' URIs we will be creating for expressions/works when basically everything is an aggregate?

Collection aggregates -- aggregating expressions/works
Augmentation aggregates -- 'main' expressions/works
Parallel aggregates -- [LC-PCC PS](https://access.rdatoolkit.org/en-US_ala-b4b70651-1768-3fd3-8342-4fae935e425e/76b6b1e5-e82e-4b9b-9720-5dd7a53a560a) says not to identify aggregating expressions; [LC-PCC MGD](https://www.loc.gov/aba/rda/mgd/mg-aggregates.pdf) says don't do it in MARC but do it in BIBFRAME

For aggregating expressions/works, I feel that too many things will go into notes except for those relationship shortcuts under Manifestation.

**Comment by GordonDunsire** [comment]
To clarify, in general a distinct description of an aggregating expression is not necessary and should be avoided.

There is a "WE" lock between an aggregating work and its aggregating expression; there is one and only one aggregating work and one and only one aggregating expression.

The aggregating expression does not inherit or reflect the characteristics of the expressions that are aggregated ("aggregated expressions"). There is little to say about the aggregating expression, and it can all be said in the description of the aggregating work. Note that an aggregating expression is, by definition, the only representative expression of an aggregating work. Note also the special instructions for aggregates in the guidance on [Representative expressions of an aggregating work](https://access.rdatoolkit.org/en-US_ala-39db8725-25e2-3bcf-8a45-7ef37ec38c90/section_ygt_gvg_3lb).

The main utility of describing a distinct aggregating expression is to cluster the expressions that are aggregated using the shortcut rdae:P20320 "is aggregated by". This utility is only required if the same set of expressions that are aggregated is embodied in multiple distinct manifestations (e.g. an aggregate in print, microfilm, and online). It is unlikely to be feasible to parse out such clusters and associated manifestations.

rdam:P30135 "has work manifested" is a shortcut that relates an aggregating work to an aggregate manifetation. So it might be safer to always create a distinct aggregating work entity for every kind of aggregate manifestation. Any expressions that are aggregated that can be parsed out may be described separately and linked to the aggregate manifestation with rdam:P30139 "has expression manifested".

**Comment by CECSpecialistI** [comment]
@pan-zhuo I am having a hard time understanding what you mean when you ask "What are the 'default' URIs we will be creating for expressions/works when basically everything is an aggregate?". Would you phrase your question another way?

**Comment by pan-zhuo** [comment]
The LC-PCC practice had me thinking we need to create 'one' expression for an aggregate manifestation, which might be for either the aggregating expression or the primary content, depending on the type of aggregate, and relate the expressions that can be identified in 7XX to it.

----
041 ## $a eng $b chi
100 10 $a Shakespeare, William, $d 1564-1616.
245 10 $a Macbeth.
380 ## $a Play
546 ## $a Text in English with summary in Chinese.

LC-PCC practice is to identify the primary expression of an augmentation aggregate, but not the aggregating expression. Then 041$b goes to notes. But if I understand @GordonDunsire's reply correctly, it seems that we could create an aggregating work, and use [P10353](http://rdaregistry.info/Elements/w/P10353) 'Work: language of representative expression' to record both 041$a and 041$b. But we still need to identify the primary work for 380, right?

----
041 ## $a chi $a eng $h eng
100 10 $a Shakespeare, William, $d 1564-1616.
245 10 $a Macbeth.
380 ## $a Play
546 ## $a Parallel texts in Chinese and English.

For a parallel aggregate, Bibframe is to create an 'aggregating expression,' and map 041$a as 'Expression: Language of expression.' It might work in Bibframe but the interpretations seem off in RDA terms, because the aggregating expression does not incorporate the content of aggregated expressions. We could bypass the aggregating expression and create an aggregating work.

----
041 ## $a chi $a eng
245 00 $a English and Chinese short stories.
380 ## $a Short stories

I think it's only feasible to create an aggregating work for a collection aggregate. But what about the 380 here? An aggregating work is a plan so it can't be a short story, right? Note on work?

----
What are the conditions that warrant the creation of an aggregating work? Presence of 041 (except when only one instance of $a), 7XX#2, or $3? And do we need to map 521 to [P10217](http://rdaregistry.info/Elements/w/P10217) 'Work: intended audience of representative expression' for an aggregate?

**Comment by GordonDunsire** [comment]
The assumption that the manifestation is an aggregate implies the statement:

thisManifestation has work manifested aggregatingWork .

Furthermore, there is only one instance of an aggregating work; there cannot be two or more aggregating works involved.

It is safe and necessary to generate (mint an IRI for) aggregatingWork.

aggregatingWork can inherit values of all or some of the characteristics that are 'representative" of the expressions that are aggregated, for the purpose of distinguishing/identifying the aggregating work from similar works.

The expressions that are aggregated do not have to be identified or described.

RDA Toolkit provides pairs of options, to cover 'all' or 'some' of the expressions that are aggregated. For example, see [this condition and options](https://access.rdatoolkit.org/en-US_ala-d69eefb5-d2bd-3dba-8faf-4090bb26ebf7/div_vpd_psf_3lb)

This approach was developed during 3R to allow policy statements to say "use this option for collection aggregates" or "use this option for the representative expression of the main work in an augmentation aggregate".

It is also useful for accommodating community string encoding schemes for the construction of access points for collection aggregating works. There was a lot of discussion about music albums; all are aggregating works that select and organize the tracks that are embodied by the manifestation (lp, cd, streaming, etc.) - the 3R core team had some knowledge (as music cataloguers, musicians, and collectors :-) plus the Aggregates Working Group was very active in providing advice on the real-world scope of the LRM model of aggregates.

Is $3 in any tag a safe indication that the manifestation is an aggregate?

It seems safe for tags that include content and exclude carrier characteristics in the definition, guidelines, and examples.

500 is not safe; the semantics cover part manifestations as well as embodied expressions.

506: example is ok, but the semantics cover multiple units ("access is to 2 binders at a time").

Issues:

1. Only one aggregating work IRI should be minted. If multiple tags in a record indicate an aggregate manifestation, then each tag must generate the same IRI and the same "work manifested" statement. In a run of the transformation software, triple statements should be de-duplicated during or after the run. One method of minting these IRIs is to add "AW" or similar to the bib record number and use it as the local part of the IRI.

2. Work-level data mapped from within such a tag is associated with triples that have the same aggregating work IRI, so they will cluster automatically to form a description set for the aggregating work.

3. Can a title of work or access point for work be generated from a bibliographic record? The starting point would be the title proper of the manifestation (245 $a). A simple template is:

access point for work = 245 $a + " (" + 260 $b + ")"

https://lccn.loc.gov/99013488_AW has access point for work "Howards End (Modern Library)" .

Note that an access point for manifestation should add 260 $c to further qualify the string, to cover reprints, etc. of the aggregate manifestation:

access point for manifestation = 245 $a + " (" + 260 $b + ", " + 260 $c + ")"

https://lccn.loc.gov/99013488 has access point for manifestation "Howards End (Modern Library, 1999)" .

4. This makes no distinction between kinds of aggregate. The distinction arises when considering what aggregated expressions/works to generate.

Collection aggregate: an aggregating work is sufficient.
Augmentation aggregate: an additional work (or expression) that is aggregated is useful if it can be determined.
Parallel aggregate: an additional work (or expression) for the work that is aggregated, and additional expressions for each language are useful if they can be determined.

**Comment by pan-zhuo** [comment]
Is it possible to safely parse out collection aggregates? If not, my guess is that we may need to map most of the work/expression level properties to notes.

041 ## $a eng
245 00 Best English short stories.
380 ## $a Short stories

**Comment by CECSpecialistI** [comment]
# Meeting Notes 2022-06-01
## Identifying aggregating and aggregated expressions
* LRM: You can't aggregate or translate aggregates
* RDA descriptions have little to say about aggregating expressions. Due to WE-lock for aggregates, we can use "representative expression of aggregating work" properties, but these are not appropriate for the conversion of legacy data and should really only be used for new descriptions.
* If more than two expressions are created as part of the transform, then we are dealing with an aggregate
* Almost everything is an aggregate :/
* When we can't identify a specific entity to which a piece of MARC data applies, we'll map to a safe property (such as note on manifestation), adding boilerplate when appropriate. 
* Main problem in expressing expression-level data in aggregates is identifying what the component expressions are of an aggregating work/expression. MARC doesn't do that. When we face this problem, we have to make the safest choices. 
* Context available in [IFLA's Final Report of the Working Group on Aggregates, including the dissenting appendix](https://www.ifla.org/wp-content/uploads/2019/05/assets/cataloguing/frbrrg/AggregatesFinalReport.pdf)
* We must follow LRM/RDA
## IRI pollution: minting lots of unnecessary/undifferentiated IRI's
* We can't assign an IRI to a thing we can't idenfity. $3 does not usually identify 
* Our goal should be to minimize entity pollution--creating potentially duplicate, mostly-empty entities
* IRI pollution creates burden on systems and reconciliation/clustering challenges
* IRI minting should be part of transformation
## Concerns surrounding lossiness/consequences of how aggregates are treated in LRM/RDA
* MARC is not structured to identify components of aggregates with enough accuracy, so a lot of the data we're concerned about losing isn't there in the first place
* How will changing the format of the data we do have, pushing 041 $a codes into string values for notes on manifestations, for instance, affect searching and discovery? Faceting by language?
* How will we map expression-to-expression relationships when aggregates are involved? 7xx's identifying translators for compilations of works, for instance?
    * Translations are parallel aggregates
    * Contributor person of text?
## Way forward
* We can leave this discussion here and revisit as we need to in order to complete the mapping
* What we are really trying to do is to identify Works, Expressions, and Manifestations--will take care not to get too far off track from that purpose

**Comment by CECSpecialistI** [comment]
Aggregate detection:
7xx second indicator 2 (1 or more)
240 $k selections
240 with conventional collective title
505 with distinct $r's or statements of responsibility separated by " / "
literature headings/subject headings intended to apply to collections
poetry--collections
free-floater "collections"?

**Reply by CECSpecialistI** [reply]
call numbers for collections

**Reply by lake44me** [reply]
On the 505, should we limit to second indicator 0 (Enhanced)?  The formatting of a "basic" (transcribed without subfields) 505 is unlikely to be parseable by punctuation with a simple algorithm.  It IS an indication of an aggregation of some kind (of chapter headings probably) but doesn't provide reliably useful data on what is aggregated for transformation.

Are we including parallel titles (indicated by punctuation with an equals sign in the 245)?  

How about, 245 contains a subfield c, and an ending period followed by more text, and that text includes a forward slash?  Or should assume that if properly cataloged, an additional title following the first statement of responsibility would also be entered as a 7xx with second indicator 2 .... ?

How about a 300 subfield e as an indication of an augmentation aggregate?  (e.g. a CD in a pocket in the back of a book about a programming language, containing exercises and code examples).

Presence of an 006 (Additional Physical Characteristics) might be a clue, but only to humans, since it can have multiple uses, some involve recording physical aspects of non-textual serials, but it might also relate to parts of serials that have different physical characteristics than the primary material (e.g. maps in textual serials).

**Reply by pan-zhuo** [reply]
It's probably not safe to base decisions on a transcribed field like 245. Any punctuation could be transcribed from the resource rather than supplied by the cataloger.

> How about, 245 contains a subfield c, and an ending period followed by more text, and that text includes a forward slash?

Both the period and the forward slash could be part of some names.

**Reply by CECSpecialistI** [reply]
Space forward slash space is what we'd be looking for if that makes a difference. I think it does. I don't think it's necessary to limit to second indicator zero if we're just using it to figure out if something is an aggregate or not. What do we do in cases where a resource has isbd punctuation in something we're transcribing? I don't think that has ever happened to me before (when spaces are accounted for)

**Reply by CECSpecialistI** [reply]
$e with same expression is not an aggregate. Presence of $e does not indicate aggregate. Meeting 2022-08-10

**Comment by AdamSchiff** [comment]
Are we including parallel titles (indicated by punctuation with an equals sign in the 245)?

No.  There are plenty of resources that have parallel titles but the content is only in one language.   Very common with Hebrew novels: title page in Hebrew, added title page in English, but the text is entirely Hebrew.  Presence of a parallel title COULD be a parallel aggregate, but not always.  You'd have to check the 041 field.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***

________________________________

**Comment by AdamSchiff** [comment]
Also very common with Russian scientific publications and also with English translations of them, which often can have both an English title and a transliterated Russian title but only English text.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
Box 352900
Seattle, WA 98195-2900
aschiff @ uw.edu
________________________________

**Comment by pan-zhuo** [comment]
I wonder how useful it is to identify certain types of augmentation aggregates.

By 'certain types' I mean aggregates with these five types of augmenting content (https://access.rdatoolkit.org/en-US_ala-cf0b18a4-5a55-3358-94b0-2d4fb5449314/div_hc5_1vv_3kb): accessibility content, colour content, illustrative content, sound content, and supplementary content.

So 300 $b illustrations AND 336 content type is one not of an image indicates an augmentation aggregate. So does 504 bibliography.

Since these contents are accounted for by the relationship shortcuts, it seems that we can treat the aggregate just as a 'singleton' and mint IRIs for the 'primary' work/expression.

It's also nearly impossible to tease out all the augmenting content in 500 (index, reader's guide, chronology, etc.).

**Comment by pan-zhuo** [comment]
$3 with a field/subfield that maps to a work/expression property

examples:

a)
245	10 $a Symphony no. 5 ; $b Divertimento no. 2 ; Machines ; Sarabande & polka ; The belles of St. Trinians / $c Malcolm Arnold.
380	## $3 The belles of St. Trinians $a Motion picture music $2 lcgft

b)
546	## $3 John P. Harrington field notebooks $a Zuni; $b Pictograms; $b Phonetic alphabet

c)
586 ## $3 Tempest fantasy $a Pulitzer prize in music, 2004

An exception is 336 content type. Could be either one expression with multiple content types or different expressions with different content types.

**Comment by CECSpecialistI** [comment]
Targeted issue specifically on conditions we will use? TG 2023-03-08

**Reply by CECSpecialistI** [reply]
https://github.com/uwlib-cams/MARC2RDA/issues/383

**Comment by briesenberg07** [comment]
📌 POSTED at uwlib-cams/sinopia_maps:  

[Request for feedback](https://github.com/uwlib-cams/sinopia_maps/discussions/180#discussioncomment-7493204) on current approaches to describing aggregate print monographs in Sinopia. See:
- [Outline of current approaches](https://github.com/uwlib-cams/sinopia_maps/discussions/180#discussion-5820104)
- [Sample description sets](https://github.com/uwlib-cams/Sandbox/tree/50a11da21530a7408421f473a6675d7cb187ea2c/sinopia_scratch/202310_aggregates)

**Reply by briesenberg07** [reply]
I've posted all the questions that I have in my head at the moment, which are many and broad. However, any feedback is appreciated--no need to address everything!

**Comment by CECSpecialistI** [comment]
Idea for Phase II: (thanks @ddanderson3 !): Run example records for aggregate types through the Phase I transformation and see what needs to change from there to prioritize fields.
